src/main/java/com/conveyal/datatools/manager/persistence/FeedStore.java

```diff
@@ -145,11 +119,11 @@ public File getFeed (String id) {
             LOG.info("Downloading feed from {}", uri);
             InputStream objectData;
             try {
-                S3Object object = s3Client.getObject(
+                S3Object object = getDefaultS3Client().getObject(
                     new GetObjectRequest(s3Bucket, key));
                 objectData = object.getObjectContent();
-            } catch (AmazonServiceException ase) {
-                LOG.error("Error downloading " + uri, ase);
+            } catch (AmazonServiceException | NonRuntimeAWSException e) {
```

Comment:
Does NonRuntimeAWSException already cover AmazonServiceException (or are they distinct)?

Reply:
They are distinct. Also, the AmazonServiceException is a RuntimeException that the compiler won't force you to handle.